From 81632d9a6b8603f6c6554c151f3fe9a66fcc7c96 Mon Sep 17 00:00:00 2001
From: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Date: Mon, 6 Nov 2017 22:53:29 +0100
Subject: [PATCH 063/448] fsl/fman: Remove a useless call to 'dev_set_drvdata()'

Commit c6e26ea8c893 ("dpaa_eth: change device used") has removed usage of
'dev_set_drvdata()' in the 'mac_probe() function.

This call should also be axed.

Signed-off-by: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/ethernet/freescale/fman/mac.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fman/mac.c b/drivers/net/ethernet/freescale/fman/mac.c
index 81021f8..7437b16 100644
--- a/drivers/net/ethernet/freescale/fman/mac.c
+++ b/drivers/net/ethernet/freescale/fman/mac.c
@@ -777,7 +777,6 @@ static int mac_probe(struct platform_device *_of_dev)
 		__devm_release_region(dev, fman_get_mem_region(priv->fman),
 				      res.start, res.end + 1 - res.start);
 		devm_kfree(dev, mac_dev);
-		dev_set_drvdata(dev, NULL);
 		return -ENODEV;
 	}
 
-- 
1.7.1

